### DISCUSSION TITLE
0.9.12

### DISCUSSION THREAD

**Discussion by Ashinch** [ReadYouApp/ReadYou#591] [Announcements]
## 0.9.12

1. Support Google Reader API and FreshRSS API (#536, @Ashinch)
2. Use forward and backward transition animation (#540, @JunkFood02)
3. Provide option to exclude additional information when exporting to OPML file (#567, @Ashinch)
4. Provide option to gray out starred article items (#547, @MauroGuida)
5. Fix "swipe to mark as read article" (#520, @aeghn)
6. Fix empty group not round bottom corner issue (#529, @kid1412621)
7. Fix refresh view after sync (#531, @kid1412621)
8. Fix error caused by missing article title (#532, @kid1412621)
9. Fix orphaned articles during sync in Fever API (#534, @Ashinch)
10. Many UI improvements and bug fixes (@Ashinch, @JunkFood02)
11. Translations updates (thanks to everyone who helped out)

### Notes

1. Thanks to the FreshRSS community, especially @Alkarex, for their help with this project.
2. ReadYou is currently not fully compatible with Inoreader through the Google Reader API.
3. ReadYou is currently unable to specify a new group when adding a subscription through the Google Reader API.
4. When syncing with Google Reader API, ReadYou can fetch up to 10k articles per sync. For read articles, only those within the past month will be synced, while unread and starred articles will be fully synced. The sync speed primarily depends on the number of these articles. The initial sync may take some time, depends on the amount of data difference between the local and server. If you have a large number of feeds, it may still feel slow, I can try to optimize this if needed.
5. We recommend that users who have participated in the testing re-sync or re-add their Google Reader API account after installing version `0.9.12`.
6. Let's look forward to more improvements in detail in the next version.

**Full changelog:** [0.9.11...0.9.12](https://github.com/Ashinch/ReadYou/compare/0.9.11...0.9.12)

<hr /><em>This discussion was created from the release <a href='https://github.com/Ashinch/ReadYou/releases/tag/0.9.12'>0.9.12</a>.</em>

**Comment by Ashinch** [comment]
#437 We might need more reports from BazQux.

**Comment by xingyucat** [comment]
Is it possible to add the ability to save pictures? Thanks.

**Reply by shuvashish76** [reply]
#372
